addons/test_event_full/tests/test_website_event.py

```diff
@@ -0,0 +1,118 @@
+        self.assertNotEqual(event.website_meta_title, view_model_track_proposal.website_meta_title, "Event and Talk Proposals page should have different meta titles")
+
+        # SEO data test of submenus
+        self.assertNotEqual(view_model_booth.website_meta_title, view_model_exhibitor.website_meta_title, "Booth page and Exhibitor page should have different meta titles")
+        self.assertNotEqual(view_model_booth.website_meta_title, view_model_community.website_meta_title, "Booth page and Community page should have different meta titles")
+        self.assertNotEqual(view_model_booth.website_meta_title, view_model_track.website_meta_title, "Booth page and Talks page should have different meta titles")
+        self.assertNotEqual(view_model_booth.website_meta_title, view_model_agenda.website_meta_title, "Booth page and Agenda page should have different meta titles")
+        self.assertNotEqual(view_model_booth.website_meta_title, view_model_track_proposal.website_meta_title, "Booth page and Talk Proposals page should have different meta titles")
+
+        self.assertNotEqual(view_model_exhibitor.website_meta_title, view_model_community.website_meta_title, "Exhibitor page and Community page should have different meta titles")
+        self.assertNotEqual(view_model_exhibitor.website_meta_title, view_model_track.website_meta_title, "Exhibitor page and Talks page should have different meta titles")
+        self.assertNotEqual(view_model_exhibitor.website_meta_title, view_model_agenda.website_meta_title, "Exhibitor page and Agenda page should have different meta titles")
+        self.assertNotEqual(view_model_exhibitor.website_meta_title, view_model_track_proposal.website_meta_title, "Exhibitor page and Talk Proposals page should have different meta titles")
+
+        self.assertNotEqual(view_model_community.website_meta_title, view_model_track.website_meta_title, "Community page and Talks page should have different meta titles")
+        self.assertNotEqual(view_model_community.website_meta_title, view_model_agenda.website_meta_title, "Community page and Agenda page should have different meta titles")
+        self.assertNotEqual(view_model_community.website_meta_title, view_model_track_proposal.website_meta_title, "Community page and Talk Proposals page should have different meta titles")
+
+        self.assertNotEqual(view_model_track.website_meta_title, view_model_agenda.website_meta_title, "Talks page and Agenda page should have different meta titles")
+        self.assertNotEqual(view_model_track.website_meta_title, view_model_track_proposal.website_meta_title, "Talks page and Talk Proposals page should have different meta titles")
+
+        self.assertNotEqual(view_model_agenda.website_meta_title, view_model_track_proposal.website_meta_title, "Agenda page and Talk Proposals page should have different meta titles")
```

Comment:
same for here also 
we can create a helper function and just pass the list of view in it it will check the uniqueness of each view 
That way it will be more clear. :smiley: